.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          cache: 'npm'
+      - run: npm ci
+      - run: sudo apt-get update && sudo apt install calibre
+      - run: npm run build
+      - env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git remote set-url origin https://git:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+          npx gh-pages \
+            --dist build/geoserver-introduction-ws \
+            --user "github-actions-bot <support+actions@github.com>" \
+            --branch gh-pages
```

Comment:
This can potentially be simplified by using the [github-pages-deploy-action](https://github.com/JamesIves/github-pages-deploy-action). See [here](https://github.com/terrestris/react-geo-ws/blob/main/.github/workflows/on-push-main.yml#L46) for an example.

Reply:
I took the this PR's approach from the [OpenLayers Workshop](https://github.com/openlayers/workshop/blob/main/.github/workflows/deploy.yml#L19-L23). I assume both approaches are valid. But if you prefer the approach with GitHub actions, just let me know. Then I will try to move it.

Reply:
Actually it's just fine.